src/components/ChatApp/MessageSection/DialogSection.js

```diff
@@ -1,21 +1,88 @@
 import Dialog from 'material-ui/Dialog';
 import React from 'react';
 import PropTypes from 'prop-types';
+import { ShareButtons, generateShareIcon } from 'react-share';
+import IconButton from 'material-ui/IconButton';
+import CloseIcon from 'material-ui/svg-icons/navigation/close';
+const { FacebookShareButton, TwitterShareButton } = ShareButtons;
+const FacebookIcon = generateShareIcon('facebook');
+const TwitterIcon = generateShareIcon('twitter');
 import Close from 'material-ui/svg-icons/navigation/close';
-
 const DialogSection = props => {
-  const closingStyle = {
-    position: 'absolute',
-    zIndex: 1200,
-    fill: '#444',
-    width: '26px',
-    height: '26px',
-    right: '10px',
-    top: '10px',
-    cursor: 'pointer',
+  const shareUrl = 'http://chat.susi.ai';
+  const title =
+    'Susi is an artificial intelligence system, combining pattern matching, internet data, data flow-, and inference engine principles. Through some abilities to reflect, it can remember the user input to produce deductions and personalized feedback. Its purpose is to explore the abilities of an artificial companion and to answer the remaining unanswered questions. The SUSI.AI web chat is a front-end developed for web access of SUSI.';
+  const style = {
```

Comment:
Move this outside the component declaration. ie. Place it between line number 10 and 11

Reply:
Also, name it style**s**. AFAIK, we have used styles in other components.